src/DZ06/ArrayValueCalculator.java

```diff
@@ -0,0 +1,61 @@
+package DZ06;
+import  java.util.logging.Logger;
+
+import static java.lang.Integer.parseInt;
+
+public class ArrayValueCalculator{
+   int summaArray;
+   int counter;
+    private static final Logger logger = Logger.getLogger(ArrayValueCalculator.class.getName());
+    static class ArraySizeException extends Exception{
+        private int a;
+        ArraySizeException(int a){
+            this.a=a;
+        }
+        @Override
+        public String toString(){
+            return ("Array list have "+this.a+" elements");
```

Comment:
You don't need variable `a`, toString. Please look to super constructor with message field